ConsoleApp9,3/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp9_3
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+
+            Console.WriteLine("Введите первое натуральное число");
+            int m = int.Parse(Console.ReadLine());
+            Console.WriteLine("Введите второе натуральное число");
+            int n = int.Parse(Console.ReadLine());
+            int ProductOfNum = n * m;
+
+            while (m != n)
+            {
+                if (m > n)
+                {
+                    m = m - n;
+                }
+                else
+                {
+                    n = n - m;
+                }
+            }
+
+            int nod = n;
+            int nok = ProductOfNum / nod;
```

Comment:
1. Используйте для имен локальных переменных стиль camelCase.
2. Если тело цикла или оператора условия состоит из одного оператора, то `{ }` можно не писать.
```suggestion
            int productOfNum = n * m;

            while (m != n)
                if (m > n)
                    m = m - n;
                else
                    n = n - m;

            int nod = n;
            int nok = productOfNum / nod;
```